app/ide-desktop/eslint.config.js

```diff
@@ -290,7 +279,7 @@ export default [
             ...tsEslint.configs.recommended?.rules,
             ...tsEslint.configs['recommended-requiring-type-checking']?.rules,
             ...tsEslint.configs.strict?.rules,
-            ...react.configs.recommended.rules,
+            ...react.configs['jsx-runtime'].rules,
```

Comment:
Starting from react 17 we don't need to import React in each component, so I replaced the `recommended` with `jsx-runtime`